tests/Unit/ProtoTests/BasicDiregapic/out/src/Gapic/LibraryServiceGapicClient.php

```diff
@@ -820,9 +819,8 @@ public function resumeOperation($operationName, $methodName = null)
      *           default this settings points to the default client config file, which is
      *           provided in the resources folder.
      *     @type string|TransportInterface $transport
-     *           The transport used for executing network requests. May be either the string
-     *           `rest` or `grpc`. Defaults to `grpc` if gRPC support is detected on the system.
-     *           *Advanced usage*: Additionally, it is possible to pass in an already
+     *           The transport used for executing network requests. At the moment, supports only
```

Comment:
It looks like the old version of this file was supporting `rest` or `grpc`, making the generated client of GRPC_REST transport type. This is very confusing, but do I get it right that `grpc+rest` as generator parameter, results in creating PHP client, which can have either `grpc` or `rest` transport specified on initialization? I.e. the `grpc` and `rest` terms are heavily overloaded here and on the moment of generation and on the moment of instantiating generated client they mean slightly different things. 

I guess we do need to have a test for rest-only generated client, but did it overwrote the other important test case - `grpc+rest` (the original version of this `LibraryServiceGapicClient.php` file)?

Reply:
The other goldens in `tests/Unit/ProtoTests/` will exercise the `grpc+rest` case. This documentation text replicates that in the [monolith PR](https://github.com/googleapis/gapic-generator/pull/3305/files), so happy to hear improvement suggestions.